dist/doc/usage.md

```diff
@@ -121,6 +123,16 @@ technology powering it.
 
 Edit this file to include any pages you need hidden from search engines.
 
+### .well-known
+
+RFC5785 [https://tools.ietf.org/html/rfc5785](https://tools.ietf.org/html/rfc5785) defines '.well-known' as a unique location for content discovery.  It contains one file - security.txt.
+
+### security.txt
+
+When security risks in web services are discovered by independent security researchers who understand the severity of the risk, they often lack the channels to disclose them properly. As a result, security issues may be left unreported. Security.txt defines a standard to help organizations define the process for security researchers to disclose security vulnerabilities securely.
```

Comment:
> When security risks in web services are discovered by independent security researchers

security issues could be discovered by any user, not just security researchers so I suggest this line be changed to remove the 'by independent security researchers'.